tests/tcl/tests/helpers/redis_queue.py

```
@@ -0,0 +1,111 @@
+        t3 = threading.Thread(target=compact, args=(client_compact, queue_name))
+        thread_list.append(t3)
+    elif mode == "auto_compact":
+        auto_compact(client_compact)
+    else:
+        print("invalid compact mode: {}".format(mode))
+        sys.exit(1)
+
+    for t in thread_list:
+        t.start()
+
+    def signal_handler(signal, frame):
```

Ruff (0.14.4)

Unused function argument: `frame`

(ARG001)